src/net.cpp

```diff
@@ -1136,6 +1152,31 @@ void CConnman::CreateNodeFromAcceptedSocket(SOCKET hSocket,
         }
     }
 
+	// Currently, users can open multiple connections to a reachable node using the same IP address and a 
+	// different port and request blockchain data at each connection. This behavior has the following impacts:
+	//   (1) Overwhelm the reachable node and consume its bandwidth by requesting data and discarding it
+	//     (see video demonstrations here: https://www.dropbox.com/s/pvasua7lnepdgfq/Occupy_Connections.mp4?dl=0
+	//     and https://www.dropbox.com/s/45ld1h7cxi3xzj6/RequestingBlockchain.mp4?dl=0)
```

Comment:
I didn't visit the website and I won't open this mp4, but a single inbound connection should be able to ask for the same data that multiple connections ask for.

Reply:
You are right. Any inbound connection should be able to request the same data that any other connection asks for. In that case, any effect on the bandwidth is perfectly fine. However, having multiple inbound connections from the same IP address makes it a lot easier for the adversary to overwhelm the bandwidth of the victim node. And this can be done easily through simple scripts without actually running a blockchain node. I have observed 33 incoming connections to my reachable node from the same IP address which raised a flag for me. 
Again, the problem circle backs to allowing multiple incoming connections from the same IP address which needs to be addressed.